.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" )
+#docker supports different storage driver, such like overlay2, aufs, etc.
+docker_storage_driver=$(docker info --format='{{.Driver}}')
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/${docker_storage_driver}" )
+stale_docker_dir_union=( "/var/lib/docker" )
+stale_kata_dir_union=( "/var/lib/vc" "/run/vc" )
+
+lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
+source "${lib_script}"
+
+metrics_lib_script="${GOPATH}/src/${tests_repo}/metrics/lib/common.bash"
+source "${metrics_lib_script}"
+
+kill_stale_process()
+{
+	# use function kill_processes_before_start() under $metrics_lib_script to kill stale containers or shim/proxy/hypervisor process
+	kill_processes_before_start
+	for stale_process in "${stale_process_union[@]}"; do
+		result=$(check_processes "${stale_process}")
+		if [[ $result -ne 0 ]]; then
+			sudo killall -9 "${stale_process}" || true
+		fi
+	done <<< "${stale_process_union}"
+}
+
+delete_stale_docker_resource()
+{
+	local docker_status=false
+	# check if docker service is running
+	systemctl is-active --quiet docker
+	if [ $? -eq 0 ]; then
+		docker_status=true
+		systemctl stop docker
```

Comment:
You may need to prefix a `sudo` here. This script may work as-is on your machine already, but some slaves may be running this as the `jenkins` user that has no direct root privs but does have passwordless `sudo`. I see you already do that for `killall`, so I guess adding here might be the right thing to do?

Reply:
+1. See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

Reply:
(although that is strictly a doc requirement we use the same assumptions "real code" too :)